test/db/extras/r2ghidra

```diff
@@ -253,6 +253,18 @@ pdg
 EOF
 RUN
 
+NAME=global variable annotation
+FILE=bins/dectest32
+EXPECT=<<EOF
+{"code":"\nundefined4 sym.get_global_array_entry(void)\n{\n    return *(undefined4 *)0x804c034;\n}\n","annotations":[{"start":1,"end":11,"type":"syntax_highlight","syntax_highlight":"datatype"},{"start":12,"end":38,"type":"function_name","name":"sym.get_global_array_entry","offset":134517184},{"start":12,"end":38,"type":"offset","offset":134517184},{"start":12,"end":38,"type":"syntax_highlight","syntax_highlight":"function_name"},{"start":39,"end":43,"type":"syntax_highlight","syntax_highlight":"keyword"},{"start":51,"end":57,"type":"offset","offset":134517193},{"start":51,"end":57,"type":"syntax_highlight","syntax_highlight":"keyword"},{"start":58,"end":59,"type":"offset","offset":134517187},{"start":60,"end":70,"type":"syntax_highlight","syntax_highlight":"datatype"},{"start":73,"end":82,"type":"global_variable","offset":134529076},{"start":73,"end":82,"type":"syntax_highlight","syntax_highlight":"constant_variable"},{"start":51,"end":82,"type":"offset","offset":134517193}]}
```

Comment:
As I noted in the PR description, I couldn't find a constant variable that satisfies all conditions. Please tell me how to find such a constant variable or give me a binary with such a constant variable. Current code checks if something is a global variable in an if condition and checks if it's constant in the next else if. I can change this order if it's required, but still, I couldn't find a constant that's in the address space "ram".

Reply:
Don't guess if you don't have an example of such variable then there is no point writing code for handling it since you don't actually know how it looks in xml or ghidra structures and how to correctly handle it.